### PR TITLE
[4.0] workflows strings

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
1. Workflows, stages and transitions are enabled/disabled not published/unpublished (just like plugins)
2. Workflows, stages and transitions can not be archived
3. Update code so we can have more specific strings such as Workflow disabled and Stage disabled instead of the generic Item disabled

### Testing Instructions
Create a new workflow and then play around with enabling/disabling/trashing the workflow, stages and transitions

### CODE REVIEW

File: administrator/components/com_workflow/Controller/StagesController.php [hunk 1]
```diff
@@ -43,6 +43,14 @@ class StagesController extends AdminController
...
 
 	/**
+	 * The prefix to use with controller messages.
+	 *
+	 * @var    string
+	 * @since  4.0.0
+	 */
+	protected $text_prefix = 'COM_WORKFLOW_STAGES';
+
+		/**
```
Comment (zero-24) [inline administrator/components/com_workflow/Controller/StagesController.php]:
```suggestion
	/**
```

File: administrator/components/com_workflow/View/Transitions/HtmlView.php [hunk 2]
```diff
@@ -179,9 +179,8 @@ protected function addToolbar()
 
 				$childBar = $dropdown->getChildToolbar();
 
-				$childBar->publish('transitions.publish');
-				$childBar->unpublish('transitions.unpublish');
-
+				$childBar->publish('transitions.publish', 'JTOOLBAR_ENABLE');
+				$childBar->unpublish('transitions.unpublish', 'JTOOLBAR_DISABLE');
```
Comment (zero-24) [inline administrator/components/com_workflow/View/Transitions/HtmlView.php:183]:
```suggestion
				$childBar->unpublish('transitions.unpublish', 'JTOOLBAR_DISABLE');

```

File: administrator/language/en-GB/en-GB.com_workflow.ini [hunk 3]
```diff
@@ -90,6 +98,16 @@ COM_WORKFLOW_TO_STAGE="Target Stage"
 COM_WORKFLOW_TRANSITION="Transition"
 COM_WORKFLOW_TRANSITIONS="Transitions"
 COM_WORKFLOW_TRANSITIONS_LIST="%s Transitions List"
+COM_WORKFLOW_TRANSITIONS_N_ITEMS_CHECKED_IN="%s transitions checked in."
+COM_WORKFLOW_TRANSITIONS_N_ITEMS_CHECKED_IN_1="%s transition checked in."
+COM_WORKFLOW_TRANSITIONS_N_ITEMS_DELETED="%s transition deleted."
+COM_WORKFLOW_TRANSITIONS_N_ITEMS_DELETED_1="%s stage deleted."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_workflow.ini]: Change `stage` to `transition`.  Repeat below.

Comment (brianteeman) [reply administrator/language/en-GB/en-GB.com_workflow.ini]: oops

**Comment by Quy** [comment]
Please update singular strings.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on fe9e71e59658ce4fc918738b8baf15bb0bb9959c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26203">issues.joomla.org/tracker/joomla-cms/26203</a>.</sub>

**Comment by brianteeman** [comment]
update branch to retrigger drone

**Comment by brianteeman** [comment]
update branch to retrigger drone

**Comment by Renuka-S** [comment]
I have tested this item :white_check_mark: successfully on 2cb7f7506204c9463e2c398d6fe7c45ebcf9786a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26203">issues.joomla.org/tracker/joomla-cms/26203</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26203">issues.joomla.org/tracker/joomla-cms/26203</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**